scripts/plop/templates/component/__tests__/Component.test.tsx.hbs

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+import { customRender } from 'test-utils';
+
+import { {{pascalCase name}} } from '../{{pascalCase name}}';
+
+describe('{{pascalCase name}}', () => {
+  it('should be tested', () => {
+    const { container } = customRender(<{{pascalCase name}}>children</{{pascalCase name}}>);
+    expect(container).toBeDefined();
```

Comment:
Funny enough, we could also fail the test to force folks to write something valid.